Patches/GTest/Patch.cmake

```diff
@@ -5,10 +5,10 @@
 # call.
 #-
 
-message("Patching GTest")
-# https://github.com/google/googletest/pull/1160
-foreach(dir googletest googlemock)
-  file(COPY ${GTest_patch}/${dir}/CMakeLists.txt DESTINATION ${GTest_source}/${dir})
-endforeach()
+# message("Patching GTest")
```

Comment:
If this is no longer needed, should we just remove it?

Reply:
Yes, I meant to do that originally but wasn't certain if the comment referring to your PR with GTest included the whole patch or just the first bit. I will double check the PR you merged and delete accordingly. I guess I should also verify that your patch did land in this release though I would be shocked if it didn't. That was about 3 years ago.

Reply:
Ah, I didn't check if the patch is *actually* no longer needed, just reacting to the patching being commented out as opposed to just dropped.